spec/integration/elasticsearch/pipeline.spec.yml

```diff
@@ -115,3 +116,7 @@ versions:
         path: /definitions/processors/if # remove rename processor validation
       - op: remove
         path: /definitions/processors/then # remove rename processor validation
+  - before: 3.3.0
+    patch:
+      - op: remove
+        path: /definitions/processors/terminate # remove terminate processor validation
```

Comment:
This array item should be added as the first one (as it references a newer version in the `before` field).

And looking at the error:
```
failed to load schema for "integration/elasticsearch/pipeline.spec.yml": failed to apply patch: error in remove for path: '/definitions/processors/terminate': unable to remove nonexistent key: terminate: missing value
```
the path should be updated too.

I think it should be like this:

```
versions:
  - before: 3.3.0
    patch:
      - op: remove
        path: /definitions/processor/properties/terminate # remove terminate processor validation
  - before: 3.1.0
    patch:
      - op: remove
        path: /definitions/processors/if # remove rename processor validation
      - op: remove
        path: /definitions/processors/then # remove rename processor validation
  
```